app/page.tsx

```diff
@@ -61,9 +68,9 @@ export default function IndexPage() {
           </CardDescription>
         </CardHeader>
         <CardContent>
-          <div className="text-lg font-semibold">Your total TOIL: 10 hours</div>
+          <CalculateTime rulesets={rulesets} hoursWorked={hoursWorked} />
           <p className="leading-7 [&:not(:first-child)]:mt-6">
-            Here&apos;s the math:
+            Here&apos;s the math: TODO
           </p>
```

Comment:
**thought:** Given we won't be working on the maths for a bit and to make it appear more complete, should we remove this TODO section and just display the total hours?

Reply:
we should add an issue to the backlog